cclib/parser/gamessparser.py

```diff
@@ -84,8 +84,16 @@ def extract(self, inputfile, line):
         """Extract information from the file object inputfile."""
         
         # extract the version number first
-        if line.find("GAMESS VERSION") >= 0:
-            self.metadata["package_version"] = line.split()[4] + line.split()[5] + line.split()[6]
+        if "Firefly version" in line:
+            match = re.search(r"Firefly version\s([\d.]*)\D*(\d*)\s*\*", line)
+            if match:
+                version, build = match.groups()
+                package_version = "{}.b{}".format(version, build)
+                self.metadata["package_version"] = package_version
+        if "GAMESS VERSION" in line:
```

Comment:
I imagine there are other things that have diverged between parsers. Do you know of any major things? Could you at least comment which match is for which program?

Reply:
I don't know. I don't actively use GAMESS and have never successfully used Firefly (some glibc or bitness problem with my machines).